cmd/queue/main.go

```diff
@@ -74,7 +74,7 @@ const (
 type config struct {
 	ContainerConcurrency     int    `split_words:"true" required:"true"`
 	QueueServingPort         string `split_words:"true" required:"true"`
-	QueueServingTLSPort      string `split_words:"true" required:"true"`
+	QueueServingTLSPort      string `split_words:"true"` // optional
```

Comment:
Upstream tests missed the upgrade issue?

Reply:
I'm still not 100% sure if this fixes the upgrade issue SO encountered. We need to verify it.

Reply:
Let's see it is a bug anyway.